internal/ocs/validations.go

```diff
@@ -0,0 +1,81 @@
+package ocs
+
+import (
+	"encoding/json"
+	"math"
+
+	"github.com/openshift/assisted-service/internal/host"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+//Info is used to return the parameters required for OCS Validations
+type Info struct {
+	CountOfOcsDisks      int64
+	CanDeployOcs         bool
+	OcsMinimalDeployment bool
+	StorageCapacity      int64
+}
+
+// ValidateOCSRequirements is used to validate min requirements of OCS
+func ValidateOCSRequirements(log logrus.FieldLogger, cluster *models.Cluster, h host.API) *Info {
```

Comment:
In the end we need to have the user see the reasons for validation failure.  Logs are only seen by Red Hat associates that are debugging the service.